### ISSUE TITLE
Crashed when using play command alongside with other command

### ISSUE THREAD

**Issue by Youki-GH** [holzschu/a-shell#721]
I ran [SOCKS Proxy Python script](https://github.com/nneonneo/iOS-SOCKS-Server) alongside with play command so that script runs in background but in less than a minute, an app crashed.
Here's the command that i used.
`python ./socks5.py | play ./silence-8hr.mp3`
Script will run in the background when i close the app. but it seems like when there is a heavy CPU usage it will crash.

Here is the crash log.
[a-Shell-mini.cpu_resource_fatal-2023-11-07-182339.txt](https://github.com/holzschu/a-shell/files/13311196/a-Shell-mini.cpu_resource_fatal-2023-11-07-182339.txt)

**Comment by holzschu** [comment]
Yes: iOS reserves the right to terminate any app that uses too much CPU, even more so if the app is in the background.
